talk/morelanguage/random.tex

```diff
@@ -0,0 +1,81 @@
+\subsection[a]{Random}
+%\epsdice[white]{3}
+
+\begin{frame}[fragile]
+  \frametitlecpp[11]{Pseudo-random number generators (PRNG)}
+  \begin{block}{Concept}
+    \begin{itemize}
+      \item For generating pseudo-random numbers the standard library offers:
+      \begin{itemize}
+        \item engines and engine adaptors to generate pseudo random bits
+        \item distributions to shape these into numbers
+        \item access to (potential) hardware entropy
+      \end{itemize}
+      \item Found in header \mintinline{cpp}{<random>}
+    \end{itemize}
+  \end{block}
+  \begin{cppcode}
+
+ \end{cppcode}
```

Comment:
```suggestion
```

Reply:
yeah, I think I wanted to add an initial example here.